File: PizzaMobileApp/pizzer_mobile/lib/models/user_info_model.dart
class UserInfoModel {
  int? id;
  String? email;
  String? name;
  String? role;
  bool? isApproved;
  String? phone;
  int? bonuses;
  String? passport;

  UserInfoModel(
      {this.id,
      this.email,
      this.name,
      this.role,
      this.isApproved,
      this.phone,
      this.bonuses,
      this.passport});

  UserInfoModel.fromJson(dynamic json) {
    id = json['id'];
    email = json['email'];
    name = json['name'];
    role = json['role'];
    isApproved = json['isApproved'];
    phone = json['phone'];
    bonuses = json['bonuses'];
    passport = json['passport'];
  }
}

File: PizzaBackendServer/WebApi/BLL/Models/UserInfoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using DAL.Data;

namespace BLL.Models
{
    public class UserInfoModel
    {
        public int Id { get; set; }
        public string Email { get; set; } = null!;
        public string Name { get; set; } = null!;
        public string Role { get; set; } = null!;
        public bool IsApproved { get; set; }
        public string Phone { get; set; }
        public int? Bonuses { get; set; }
        public string? Passport { get; set; }
        public string? ApprovalCode { get; set; }

        public UserInfoModel() { }
        public UserInfoModel(UserModel u)
        {
            Id = u.Id;
            Email = u.Email;
            Name = u.Name;
            Role = u.Role;
            IsApproved = u.IsApproved;
            Phone = u.Phone;
            Passport = u.Passport;
            Bonuses = u.Bonuses;
            ApprovalCode = u.ApprovalCode;
        }
    }
}